### PR TITLE
Add javascript-hooker@0.2.3 with git autoupdate

### PR DESCRIPTION
**Pull request by LeaYeh**
**git repo url:** https://github.com/cowboy/javascript-hooker/tree/master/dist
**Watch 7
 Star 157
 Fork 9**
@maruilian11  could you help me check out the PR for #6201  ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/javascript-hooker/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "homepage": "http://github.com/cowboy/javascript-hooker",
+  "author": {
+    "name": "\"Cowboy\" Ben Alman",
+    "url": "http://benalman.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/cowboy/javascript-hooker.git"
+  },
+  "licenses": [
```
Comment (maruilian11) [inline ajax/libs/javascript-hooker/package.json]: Not need an array here~

File: ajax/libs/javascript-hooker/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
```
Comment (PeterDaveHello) [inline ajax/libs/javascript-hooker/package.json]: No need to use an object here...

**Comment by LeaYeh** [comment]
@maruilian11 ok, i got that

**Comment by maruilian11** [comment]
@LeaYeh peter said if the lib has npm package, we prefer to use npm auto-update.
sorry, didn't mention it last time 😓

**Comment by LeaYeh** [comment]
that's fine, i fix and rebase it. check again

**Comment by LeaYeh** [comment]
@maruilian11 i fixed the problem and rebase the commit, could you check it again?

**Comment by PeterDaveHello** [comment]
Where is the npmName config?

**Comment by LeaYeh** [comment]
@PeterDaveHello oops sorry i missed that field. fixed and rebased it.

**Comment by PeterDaveHello** [comment]
```
npm ERR! Linux 3.13.0-76-generic
npm ERR! argv "/home/peter/.nvm/versions/node/v4.2.6/bin/node" "/home/peter/.nvm/versions/node/v4.2.6/bin/npm" "i" "javascript-hooker"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/javascript-hooker
npm ERR! 404
npm ERR! 404 'javascript-hooker' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /run/shm/npm-debug.log

```

**Comment by LeaYeh** [comment]
@PeterDaveHello i added npmName follow [this](https://www.npmjs.com/package/hooker) is it still going wrong?

**Comment by PeterDaveHello** [comment]
You added a wrong name from previous commit.

**Pull request merged**